File: SoenWebApi/docs/enviar-texto.js
#!/usr/bin/env node
/** Exemplo: enviar mensagem de texto via API */
const API = process.env.API_URL || 'http://localhost:3000';
const KEY = process.env.API_KEY || '';

const num = process.argv[2] || '5511999999999';
const msg = process.argv[3] || 'Ola, tudo bem?';

const opts = {
  method: 'POST',
  headers: { 'Content-Type': 'application/json' },
  body: JSON.stringify({ number: num, message: msg }),
};
if (KEY) opts.headers['x-api-key'] = KEY;

fetch(`${API}/api/send/text`, opts)
  .then(r => r.json())
  .then(console.log)
  .catch(console.error);

File: SoenWebApi/docs/enviar-texto.py
#!/usr/bin/env python3
"""Exemplo: enviar mensagem de texto via API"""
import requests, sys

API = "http://localhost:3000"
KEY = ""  # ou leia de .env

num = sys.argv[1] if len(sys.argv) > 1 else "5511999999999"
msg = sys.argv[2] if len(sys.argv) > 2 else "Ola, tudo bem?"

hdr = {"x-api-key": KEY} if KEY else {}
r = requests.post(f"{API}/api/send/text", json={"number": num, "message": msg}, headers=hdr)
print(r.json())
